src/pages/portfolio/EquityTiers.tsx

```diff
@@ -0,0 +1,163 @@
+                  nextLevelRequiredTotalNetCollateralUSD,
+                }) => (
+                  <$NetCollateralValue>
+                    {requiredTotalNetCollateralUSD > 0 && (
+                      <>
+                        <span>{`≥ `}</span>
+                        <$HighlightOutput
+                          type={OutputType.CompactFiat}
+                          value={requiredTotalNetCollateralUSD}
+                        />
+                      </>
+                    )}
+
+                    {requiredTotalNetCollateralUSD > 0 &&
+                      nextLevelRequiredTotalNetCollateralUSD && (
+                        <$AndText>{stringGetter({ key: STRING_KEYS.AND })}</$AndText>
+                      )}
+
+                    {nextLevelRequiredTotalNetCollateralUSD && (
+                      <>
+                        <span>{`< `}</span>
+                        <$HighlightOutput
+                          type={OutputType.CompactFiat}
+                          value={nextLevelRequiredTotalNetCollateralUSD}
+                        />
+                      </>
+                    )}
+                  </$NetCollateralValue>
+                ),
+              },
+              {
+                columnKey: 'maxOrders',
+                label: stringGetter({ key: STRING_KEYS.LONG_TERM_OR_CONDITIONAL_ORDERS }),
+                allowsSorting: false,
+                renderCell: ({ maxOrders }) => (
+                  <$HighlightOutput type={OutputType.Number} value={maxOrders} />
+                ),
+              },
+            ] as ColumnDef<EquityTier>[]
+          ).filter(isTruthy)}
+          selectionBehavior="replace"
+          paginationBehavior="showAll"
+          withOuterBorder={isNotTablet}
+          withInnerBorders
+        />
+      </$ContentWrapper>
+    </AttachedExpandingSection>
+  );
+};
+const $ContentWrapper = styled.div`
+  ${layoutMixins.flexColumn}
+  gap: 1.5rem;
+  max-width: 100vw;
+`;
+
+const $Table = styled(Table)`
+  --tableCell-padding: 0.5rem 1.5rem;
+  --bordered-content-border-radius: 0.625rem;
+  --table-cell-align: end;
+
+  font: var(--font-base-book);
+
+  @media ${breakpoints.mobile} {
+    --tableCell-padding: 1rem 1.25rem;
+    font: var(--font-small-book);
+  }
+
+  @media ${breakpoints.notTablet} {
+    --tableStickyRow-backgroundColor: var(--color-layer-1);
+  }
+` as typeof Table;
+
+const $HighlightOutput = styled(Output)`
+  color: var(--color-text-1);
+`;
+
+const $NetCollateralValue = styled.span`
+  ${layoutMixins.inlineRow}
+`;
+
+const $AndText = styled.span`
+  color: var(--color-text-0);
+`;
+
+const $Description = styled.div`
+  color: var(--color-text-0);
+  --link-color: var(--color-text-1);
+  padding: 0 1rem;
+  font: var(--font-small-book);
+
+  @media ${breakpoints.tablet} {
+    padding: 1.25rem 1.5rem 0;
+  }
+
+  span {
+    ::after {
+      content: ' ';
+    }
+  }
+
+  a {
+    display: inline;
```

Comment:
should links be inline-grid?

Reply:
is inline-grid preferred when the links have multiple elements in it? it feels like in this case it's just text so inline should be fine? no strong preference tho

Reply:
I was just noting that the internal styles for <Link> expect to be a grid and I've noticed in other places we try to respect that and make it inline-grid if we want it inline. If it renders fine I don't mind either way though!